src/devices/bus/segaai/segaai_exp.h

```diff
@@ -0,0 +1,61 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol
+#ifndef MAME_BUS_SEGAAI_EXP_H
+#define MAME_BUS_SEGAAI_EXP_H
+
+#pragma once
+
+
+DECLARE_DEVICE_TYPE(SEGAAI_EXP_SLOT, segaai_exp_slot_device);
+
+
+class segaai_exp_slot_device : public device_t,
+								public device_slot_interface
+{
+public:
+	template <typename T>
+	segaai_exp_slot_device(const machine_config &mconfig, const char *tag, device_t *owner, T &&opts, const char *dflt)
+		: segaai_exp_slot_device(mconfig, tag, owner, u32(0))
+	{
+		option_reset();
+		opts(*this);
+		set_default_option(dflt);
+		set_fixed(false);
+	}
+	segaai_exp_slot_device(const machine_config &mconfig, const char *tag, device_t *owner, u32 clock);
+	virtual ~segaai_exp_slot_device() { }
```

Comment:
Please move this destructor out-of-line.

Reply:
Fixed.